hash/hash.go

```diff
@@ -0,0 +1,119 @@
+package hash
+
+import (
+	"bytes"
+	"crypto/subtle"
+	"encoding/base64"
+	"fmt"
+	"strings"
+
+	"gitlab.com/NebulousLabs/errors"
+	"gitlab.com/NebulousLabs/fastrand"
+	"golang.org/x/crypto/argon2"
+)
+
+var (
+	ErrInvalidHash               = errors.New("the encoded hash is not in the correct format")
```

Comment:
Follow Up: there must be some linting that this go repo doesn't have. skyd complains if exported items don't have comments.

Reply:
F/U: https://github.com/SkynetLabs/skynet-accounts/issues/64